### ISSUE TITLE
Feature Request: Extend the usage of exclude_paths

### ISSUE THREAD

**Issue by hbokh** [warpnet/salt-lint#100]
Hello Warpnet,

We have a lot of Jinja2-files in subdirectories of states and sometimes a bit deeper too.
These directories do not have `.sls` files in them.
So running `salt-lint` gives warnings like this (combined output from multiple runs):
```
WARNING: Coudn't open salt/postfix/files/init.sls - No such file or directory
WARNING: Coudn't open salt/mysql/files/init.sls - No such file or directory
WARNING: Coudn't open salt/os/debian/files/init.sls - No such file or directory
...
etc.
```

Would it be possible to extend the `exclude_paths` with something like this?

```
---
exclude_paths:
  - salt/*/files/*
skip_list:
...
```
Thanks in advance!

**Comment by roaldnefs** [comment]
Thanks @hbokh for your feature request! That would be possible by using `pathspec` library, which is already a dependency of `salt-lint`. I will try to fix this in the next release.

**Comment by hbokh** [comment]
Great, Roald! While you’re at it, add an “l” to `WARNING: Coudn't open ...`. 😇

**Comment by hbokh** [comment]
On the other hand: maybe it is better to always run something like `salt-lint $(find salt pillar -name '*.sls')` instead of `salt-lint salt/postfix/*`... 🤔

**Comment by roaldnefs** [comment]
Already fixed in #126, closing this issue.
